js/ui/data_grid.d.ts

```diff
@@ -4353,17 +4353,17 @@ declare class dxDataGrid<TRowData = any, TKey = any> extends Widget<dxDataGridOp
     /**
      * @docid
      * @publicName getSelectedRowKeys()
-     * @return Array<any> | Promise<any>
+     * @return Array<any> & Promise<any>
```

Comment:
Actually it's a union, right? We do not provide an object that can used as an `Array` and `Promise` at the same time
```suggestion
     * @return Array<any> | Promise<any>
```